psi4/driver/procrouting/proc.py

```diff
@@ -427,26 +427,29 @@ def select_mp3_gradient(name, **kwargs):
     reference = core.get_option('SCF', 'REFERENCE')
     mtd_type = core.get_global_option('MP_TYPE')
     module = core.get_global_option('QC_MODULE')
+    all_electron = core.get_global_option('FREEZE_CORE') == "FALSE"
```

Comment:
Could we have `all_electron = (core.get_global_option('FREEZE_CORE') == "FALSE")`? It's more legible...

Reply:
done!